inc/Classes/Auth.php

```diff
@@ -319,6 +319,15 @@ public function login($email, $password, $show_confirmation = 1)
 
             // Everything fine!
             } else {
+                if ($user["user_login"] and PasswordHash::needsRehash($user["password"])) {
+                    try {
+                        $db->qry('UPDATE %prefix%user SET password = %string% WHERE userid = %int%', PasswordHash::hash($password), $user["userid"]);
+                        $func->information(t('Es wurde ein Sicherheitsupgrade von deinem Passwort durchgeführt.'), '', 1);
+                    } catch (Exception $e) {
```

Comment:
```suggestion
                    } catch (\Exception $e) {
```

Add proper Root-Namespace to exception.